src/tools/wasm-link.cc

```diff
@@ -695,6 +704,16 @@ static void calculate_reloc_offsets(Context* ctx) {
     binary->imported_function_index_offset = total_function_imports;
     binary->imported_global_index_offset = total_global_imports;
     binary->memory_page_offset = memory_page_offset;
+
+    size_t delta = 0;
+    for (size_t i = 0; i < binary->function_imports.size(); i++) {
+      if (!binary->function_imports[i].active) {
+        delta++;
```

Comment:
else here since we don't want to store relocated_function_index for inactive imports?